htdocs/web_portal/views/site/edit_api_auth.php

```diff
@@ -67,6 +92,16 @@
                 <label class="input_label" for="ALLOW_WRITE">Allow API write</label>
             </div>
         </div>
-        <input type="submit" value="Edit credential" class="input_button">
+        <?php } ?>
+
+        <?php if ($params['isRenewalRequest']) {
+            echo '<br><p> Are you sure you want to continue? </P>';
+            echo '<input class="input_input_hidden" type="hidden" name="isRenewalRequest" value=true />';
```

Comment:
can this line be broken up?

Reply:
As discussed, adhering to limit now. Done